pkgs/development/python-modules/qtwidgets/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pyqt5,
+  setuptools-scm,
+}:
+buildPythonPackage {
+  pname = "qtwidgets";
+  version = "1.2";
```

```suggestion
  version = "1.2-unstable-YYYY-MM-DD";
```
The version is from a commit so you need to put the date here.

---

Done.